src/pybind/mgr/cephadm/serve.py

```diff
@@ -79,6 +81,10 @@ def serve(self) -> None:
 
                 self._update_paused_health()
 
+                if self.mgr.need_connect_dashboard_rgw:
+                    self.mgr.need_connect_dashboard_rgw = False
+                    self.mgr.cephadm_services['rgw'].connect_dashboard_rgw()
```

Comment:
```suggestion
                if self.mgr.need_connect_dashboard_rgw and self.mgr.config_dashboard::
                    self.mgr.need_connect_dashboard_rgw = False
                    self.mgr.cephadm_services['rgw'].connect_dashboard_rgw()
```

Why duplicate 

https://github.com/ceph/ceph/blob/9ab9cc26e200cdc3108525770353b91b3dd6c6d8/src/pybind/mgr/cephadm/services/cephadmservice.py#L198-L204

here?